pages/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/kudo-cassandra/index.md

```diff
@@ -0,0 +1,214 @@
+
+#include /dkp/kommander/1.3/include/kudo-cassandra-instance.tmpl
+
+- Get the Plan status from the KUDO Cassandra Operator.
+
+#include /dkp/kommander/1.3/include/kudo-cassandra-plans.tmpl
+
+- A `deploy` plan status of `COMPLETE` indicates that KUDO Cassandra has deployed successfully and is healthy.
+
+If any issues are encountered during the above, refer to the [Troubleshooting](#Troubleshooting) section.
+
+### Available Parameters
+
+The complete list of KUDO Cassandra Parameters can be found under [detailed parameter descriptions](https://github.com/mesosphere/kudo-cassandra-operator/blob/master/docs/parameters.md).
+
+
+The current parameters set can be reterived using the kubectl command with the two additional tools:
+- [jq](https://stedolan.github.io/jq)
+- [yq](https://mikefarah.gitbook.io/yq)
+
+To retrive the current parameters, issue the following command in the terminal with appropriate `INSTANCE` value set:
+```
+INSTANCE=cassandra;
+kubectl get instances -o json | jq ".items[] | select(.metadata.name == \"$INSTANCE\") | .spec.parameters" | yq -e --yaml-output '.' > cassandra-params.yml
+```
+
+The above command generates a file called `cassandra-params.yml` with the current values of all the parameters in effect for the `cassandra` instance.
+
+### Updating Parameters
+
+Parameters can be updated using arguments to the KUDO CLI.
+
+**Example**: Increasing Cassandra node counts
+- Increase the number of nodes using the KUDO CLI:
+```
+kubectl -n test-project-zc6tc kudo update --instance cassandra -p NODE_COUNT=4
+```
+- Monitor the KUDO Cassandra deployment plan:
+```
+kubectl kudo plan status --instance cassandra -n test-project-zc6tc
+```
+- Wait for the deployment plan to have a status of `COMPLETE`
+
+
+When the deployment plan is `COMPLETE` there should be 4 nodes as seen by the number of pods running:
+```
+kubectl get pods -n test-project-zc6tc
+NAME               READY   STATUS    RESTARTS   AGE
+cassandra-node-0   1/1     Running   0          62m
+cassandra-node-1   1/1     Running   0          62m
+cassandra-node-2   1/1     Running   0          61m
+cassandra-node-3   1/1     Running   0          8m13s
+```
+
+**Example**: Updating multiple parameters:
+
+To update multiple parameters at once, it is recommended to submit the updated parameters using the KUDO CLI.
+
+See [Available Parameters](#available-parameters) to get the full list of current parameters as a file.
+
+Apply the desired updates in `cassandra-params.yml` using the KUDO CLI:
+```
+kubectl kudo update -n test-project-zc6tc --instance=cassandra -P cassandra-params.yml 
+```
+Wait for the deployment plan to `COMPLETE` as shown in the Cassandra node counts example.
+
+### Upgrades
+
+KUDO Cassandra versions can be upgraded using the KUDO CLI.
+
+**Example** Upgrade KUDO Cassandra from `v3.11.6-1` to `v3.11.7-1`:
+```
+kubectl kudo upgrade cassandra -n test-project-zc6tc --instance cassandra --operator-version 1.0.2 
+```
```

Comment:
At first glance, it is not at all clear what `operator version 1.0.2` has to do with Cassandra `v3.11.7-1`.  I understand it _after_ reading the Cassandra operator docs page (which has a table), should there be a reference to that table here?  Also, in that table,
the versions are listed as `3.11.7-1.0.2` but here you have `v3.11.7-1`.  Which is correct?

Here's my suggested fix for these:  

```suggestion
**Example** Upgrade KUDO Cassandra from `v3.11.6-1.0.1` to `v3.11.7-1.0.2`:
When upgrading, you should understand the mapping between Cassandra versions and operator versions.   For more information, see the table at the end of the [Cassandra operator](https://github.com/mesosphere/kudo-cassandra-operator/tree/master/docs)
```
kubectl kudo upgrade cassandra -n test-project-zc6tc --instance cassandra --operator-version 1.0.2 
```
```